legacy/package.json

```diff
@@ -86,7 +86,7 @@
     "js-cookie": "3.0.1",
     "ng-tags-input": "3.2.0",
     "single-spa-angularjs": "4.3.1",
-    "vanilla-framework": "2.37.1"
+    "vanilla-framework": "3.0.0-rc.1"
```

Comment:
We should use the latest stable instead of a release candidate, but this can be done in another PR.

Reply:
Oh no, thanks. It only affected the controller details and the initial loading screen, so I've upgraded to the release versions for both and it looks fine to me.

Reply:
Ok great, even better. Thanks!